src/pages/ProjectSettings.tsx

```diff
@@ -110,7 +110,7 @@ export const ProjectSettings: React.VFC = () => {
 
   return (
     <ProjectSettingsProvider>
-      <SideNav aria-label="Project Settings">
+      <SideNav aria-label="Project Settings" widthOverride={250}>
```

Comment:
Should we make this consistent everywhere else? (Maybe a question for design)

Reply:
I think the only reason the project settings page needs it is because of the dropdown. The project settings page also has longer section titles. It would not look good on our other SideNavs (see how it just takes away from the table space):

<img width="1570" alt="Screen Shot 2022-12-06 at 4 58 54 PM" src="https://user-images.githubusercontent.com/47064971/206032230-24aea66b-014a-4ced-b1e9-e62c14139d39.png">